packages/core/admin/server/config/api-token.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = ({ env }) => ({
+  salt: env('API_TOKEN_SALT'),
+});
```

Comment:
I think you have to declare an object, not a function. If you run your app twice, it will throw the error "there's something wrong with the configuration of your api-token salt."

Reply:
I think I got confused with the config files that are present in the plugins 🤔 

Will fix that.

Reply:
Understandable :p